docs/source/fft.rst

```diff
@@ -0,0 +1,21 @@
+.. role:: hidden
+    :class: hidden-section
+
+torch.fft
+===================================
+
+Discrete Fourier transforms and related functions.
+
+To use these functions the torch.fft module must be imported since its name
+conflicts with the :func:`torch.fft` function.
```

Comment:
I'm not necessarily recommending this as it's definitely a hack, but you can workaround this issue. Change the `fft` module's `__class__` with a class that has a static `__call__` method defined. We used this when adding `scipy.fft`:
https://github.com/scipy/scipy/blob/4e3d82268eae3c89e94ff5f3c3774bb12c12982a/scipy/fft/__init__.py#L101-L115

Reply:
Hey @peterbell10! That's a good idea and I tried it but it's not currently jittable. I'm following up with the JIT team on the possibility of making that work with the jit, but I don't expect that to be a reasonable solution.

Reply:
(Also, how much of a hack is it when Guido himself tells us to do it?)

Reply:
It breaks things like sphinx autodoc which try to do inspection on module which isn't great. I certainly wouldn't design an API like that intentionally.

Reply:
We may be able to do something like this in the future after some JIT work.